bHY6yh-de6884/cod-bxxqqq_bHY6yh-de6884.xml

```diff
@@ -0,0 +1,317 @@
+            </p>
+            <p xml:id="bHY6yh-de6884-xxx432">   
+              / <ref xml:id="bHY6yh-de6884-Rxx32">Idem
+              <lb ed="#B"/><unclear>dt</unclear>Marcus 8 capitulo</ref>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e204">
+              ¶ <ref>18 capitulo</ref> <quote>ve mundo
+              <lb ed="#B"/>ascandalis</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e213">
+              ¶ <ref xml:id="bHY6yh-de6884-Rd1e220">19 capitulo</ref> <quote xml:id="bHY6yh-de6884-Qd1e215">Ait domini si 
+              <lb ed="#B"/>vis perfectus esse vade et vende 
+              <lb ed="#B"/>omnia que habes. et da pauperibus 
+              <lb ed="#B"/>et habebis thesaurum in celis et veni 
+              <lb ed="#B"/>sequere me</quote> 
+            </p>
+            <p xml:id="bHY6yh-de6884-xxx342">
+              ¶ <ref xml:id="bHY6yh-de6884-Rxx342">dixit dominis discipulis 
+              <lb ed="#B"/>suis</ref> <quote xml:id="bHY6yh-de6884-Qxx342">Amen dico vobis quia diues
+              <lb ed="#B"/>difficile intrabit regnum celorum 
+              <lb ed="#B"/>Et iterum dico vobis facilius est 
+              <lb ed="#B"/>camelum perfer anime acus transi<lb ed="#B" break="no"/>re 
+              quam diuitem intrare in regnum
+              <lb ed="#B"/>celorum Auditis ergo hiis discipuli 
+              <lb ed="#B"/>mirabantur valde dicentes quis 
+              <lb ed="#B"/>ergo peroterit saluus esse Aspiciens 
+              <lb ed="#B"/>autem ihesus dixit illis Apud hominem 
+              <lb ed="#B"/>hoc impossibile est apud deum autem
+              <lb ed="#B"/>omnia possibilia sunt</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e242">
+              ¶ <ref xml:id="bHY6yh-de6884-Rd1e251">Idem dt lucas 
+              <lb ed="#B"/>in ewangelo capitulo 18</ref> <quote xml:id="bHY6yh-de6884-Qd1e244">Ait ihesus omnis qui 
+              <lb ed="#B"/>reliquit domum uel frens aut sorores 
+              <lb ed="#B"/>aut patrem aut uxorem aut matrem aut
+              <lb ed="#B"/>filios aut agros propter nomen meum 
+              <lb ed="#B"/>centuplum accipiet et vitam eternam 
+              <lb ed="#B"/>possidebit</quote>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e258">
+              ¶ <ref>Idem dicit lucas in ewangelo
+              <lb ed="#B"/>capitulo 18</ref>
+            </p>
+            <p xml:id="bHY6yh-de6884-d1e266">
+              <ref xml:id="bHY6yh-de6884-Rd1e280">Marcus in ewangelo capitulo 8</ref> <quote xml:id="bHY6yh-de6884-Qd1e270">Ihesus 
+              <lb ed="#B"/>cepit docere discipulos suos quoniam 
+              <lb ed="#B"/>oportet filium hominis multa pati et repro<lb ed="#B" break="no"/>bari 
+              a senioribus et a summis sacer<lb ed="#B" break="no"/>dotibus 
+            <pb ed="#B" n="12-v"/>
+            <cb ed="#B" n="a"/>
+              et scribis et occidi et post 
```

Comment:
If this is the beginning of a new column, then there should also be the beginning of new line, no?

```suggestion
            <cb ed="#B" n="a"/>
            <lb ed="#B"/>et scribis et occidi et post 
```